dev/doc/debugging.md

```diff
@@ -47,7 +47,7 @@ Debugging with ocamldebug from Emacs
    7. some hints: 
 
    - To debug a failure/error/anomaly, add a breakpoint in
-     Vernac.vernac_com at the with clause of the "try ... interp com
+     `Vernac.interp_vernac` (in `toplevel/vernac.ml`) at the with clause of the "try ... interp com
```

Comment:
This is ok but actually not exclusive.

These days, I'm more putting a breakpoint in the `reraise` of `Vernacentries.interp` (around the end of file). This reduces the number of `try with` to go up (even though there are still some with a breakpoint in `Vernacentries.interp`).

When failures come from a tactic call, it is a bit more difficult. Sometimes, I put a breakpoint in `Proofview.tclZERO` and this saves some steps wandering in the proof engine. I should try also `Proofview.wrap_exceptions` which, often, is just one step before the actual error is raised. However, I'm not sure that every proof engine error is going through `wrap_exceptions`, so, in last analysis, `Vernacentries.interp` is the surer point to catch any kind of error or anomaly.

Don't know the experience of others though.